### DISCUSSION TITLE
There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

### DISCUSSION THREAD

**Discussion by HikaruChang** [community/community#65446] [GitHub Education]
### Hi everyone,

My account was suddenly prompted for these contents.

There are outstanding issues with your GitHub account. Please contact [GitHub Support](https://support.github.com/contact?tags=education-support) for more information.

After I tried to initiate a ticket, no staff has contacted me for over 3 weeks.

**Comment by zhangchi25806** [comment]
Same here.
Tried:

- [x] Submit a ticket
- [x] Create a thread here

Nothing helps even a bit

**Reply by HikaruChang** [reply]
I'm not sure what the hell GitHub is up to, maybe tired of student users?

**Comment by zhangchi25806** [comment]
> I'm not sure what the hell GitHub is up to, maybe tired of student users?

I think they still cared because they blocked us LOL
